lapis/simulator.py

```diff
@@ -138,7 +138,7 @@ def create_storage(
     def create_scheduler(self, scheduler_type):
         self.job_scheduler = scheduler_type(job_queue=self.job_queue)
 
-    def create_connection_module(self, remote_throughput, filebased_caching=True):
+    def create_connection_module(self, remote_throughput, filebased_caching=False):
```

Comment:
What is the purpose of changing the default value here from `True` to `False`? Will this impact experiments we do based on the configurations provided by Tabea? In case of yes, we should leave it as is for reproducibility.

Reply:
Tabea set the default in her scripts to `False`. To avoid confusion I thought it might be safer to get rid of the additional layer and set it to `False` already here, instead of setting a default value for the flag in the CLI.

Reply:
I think as long as nothing relies-on, it's fine. In the long run I tend to completely remove that flag.